chalicelib/checks/wrangler_checks.py

```diff
@@ -2412,19 +2412,29 @@ def compare_record(existing_record, user, all_labs, all_grants, new=False):
                 updates[a_key] = a_val
         return updates
 
-    def find_lab(record, all_labs):
+    def find_lab(record, all_labs, all_grants):
         lab = ''
         all_lab_names = [i['display_title'] for i in all_labs]
         score = 0
         best = ''
+        log = []
         for disp in all_lab_names:
             s = fuzz.token_sort_ratio(record['OH Lab'], disp.split(',')[0])
             if s > score:
                 best = disp
                 score = s
         if score > 73:
             lab = [i['@id'] for i in all_labs if i['display_title'] == best][0]
-        return lab, score
+
+        if not lab:
+            oh_grant = record.get('OH Grant', '')
```

Comment:
So here you are getting a value from the OH spreadsheet, right?  And if there isn't one then oh_grant is the empty string.